File: composeApp/src/commonMain/kotlin/com/hhp227/concafe/presentation/cafe/event/CafeEventAction.kt
package com.hhp227.concafe.presentation.cafe.event

sealed interface CafeEventAction {
    data object ClickBack : CafeEventAction
    data object Retry : CafeEventAction
    data object ToggleLike : CafeEventAction
    data object GoToCafe : CafeEventAction
    data class ClickCast(val castId: String) : CafeEventAction
    data object ClickLoginPromptSignIn : CafeEventAction
    data object DismissLoginPrompt : CafeEventAction
}

File: iosApp/iosApp/Presentation/Cafe/Event/CafeEventAction.swift
//
//  CafeEventAction.swift
//  ConCafe
//
//  Created by 홍희표 on 4/23/26.
//

import Foundation

enum CafeEventAction {
    case backTapped
    case retry
    case toggleLike
    case goToCafe
    case castTapped(id: String)
    case loginPromptSignInTapped
    case dismissLoginPrompt
}
